RemoveLinksToAttachments.py

```diff
@@ -2,11 +2,14 @@
 Redmine Support #4173
```

Comment:
remove the link to the redmine support ticket from the script itself

Reply:
Banner has been rewritten to remove the redmine support ticket